src/containers/DashRoutes.tsx

```diff
@@ -39,6 +40,7 @@ function DashRoutes() {
           <Route path="/cohorts" element={<AdminCohorts />} />
           <Route path="/sessions" element={<AdminSession />} />
           <Route path="/manage" element={<AdminManageRoles />} />
+          <Route path="/grading-system" element={<GradingSystem />} />
```

Comment:
Ideally, paths should be short as long as they are descriptive enough. Here, I believe it's safe to omit system and stick with `/grading`